dll/cpl/desk/settings.c

```diff
@@ -842,6 +842,35 @@ SwitchDisplayMode(HWND hwndDlg, PWSTR DeviceName, PSETTINGS_ENTRY seInit, PSETTI
     }
 }
 
+static
+PSETTINGS_ENTRY
+FindBestElement(
+    _In_ PDISPLAY_DEVICE_ENTRY pDevice)
+{
+    PSETTINGS_ENTRY Current, Request = &pDevice->InitialSettings, List = pDevice->Settings;
+    PSETTINGS_ENTRY pBestEntry = NULL;
+    LONG Distance, NearestDistance = MAXLONG;
+
+    /* Find the best entry in the list */
+    for (Current = List; Current; Current = Current->Flink)
```

Comment:
```suggestion
    PSETTINGS_ENTRY Request = &pDevice->InitialSettings, pBestEntry = NULL, Current;
    LONG Distance, NearestDistance = MAXLONG;

    /* Find the best entry in the list */
    for (Current = pDevice->Settings; Current; Current = Current->Flink)
```

--

And you may `s/pBestEntry/BestEntry/g`.